### ISSUE TITLE
Engage readers

### ISSUE THREAD

**Issue by ericlebihan** [lesintegristes/lesintegristes-theme#34]
Solicit input from readers to improve the blog; It could be open or closed questions.

For example 

Closed question : 

In the past we have done several translations of english articles, would you be interested by foreign articles translated in french ? (Yes/No)

Open question :

How would you want lesintegristes to be improved in the near future, what are the fonctionnalities you would find useful ?

With interactions like : discussions / contributions /  improve the blog I think we could make lesintegristes a lot more alive than it is today.

What do you think ?

**Comment by gregoiredierendonck** [comment]
Personally, I think it's a good idea.

I'd proposed to rename the "Note" by "Discussions" for example (With opened/closed questions). It was a bad idea.
But I think it's the good way to directly engaging readers...
Do we need a new entrie in the menu for that? Complicated...

What do you mean exactly, @ericlebihan?

**Comment by ericlebihan** [comment]
@gregoiredierendonck 

What do you mean when you say "what do you mean exactly  ?" :-)
Could you be more specific ?

**Comment by gregoiredierendonck** [comment]
Sorry, wasn't really clear... :/

I meaned: _**where** do you want to input this question?_
Direct in the Note, in the article? In a specific page in the site? In a specific category?
I suggested creating a place/a page to make it, and to create a more engaged discussion inside of the blog.
(It's just an idea, I don't know if it good is or not... :) )

**Comment by ericlebihan** [comment]
Yes I meant a special page, but we have to figure out the interactions between the notes / articles pages and the contribute page, and see if we need to make another page "improve the blog" or keep it in the contribute page. What do you think ?
